### DISCUSSION TITLE
Upgraded from Wix 3 to 5 and now product version isn't displaying the same

### DISCUSSION THREAD

**Discussion by jackalford2** [wixtoolset/issues#8701] [Questions] [answered]
We recently upgraded a stack of projects from Wix 3 to Wix 5 and now if we look at the .msi's using Orca the ProductVersion isn't displaying the same value that's in the Product.wxs file. Is there a switch somewhere that is controlling this behavior or have we missed something in going from V3 to V5?

Version value in Product.wxs

<Package Language="1033" Version="**17.00.01**" 

What I see using Orca

ProductVersion    **17.0.1**

Thanks for any assistance.

Jack

**Comment by wmanning** [comment]
Yes it did change to get rid of the leading zeros.  Can't seem to find the discussion (or issue?) now.

**Reply by barnson** [reply, marked as answer]
https://github.com/wixtoolset/issues/issues/8589

**Reply by jackalford2** [reply]
Thanks @barnson for the link to the issue. I guess I didn't know what to search for.
